Dockerfile

```diff
@@ -15,12 +15,18 @@ RUN addgroup --gid 2000 --system appgroup && \
 WORKDIR /app
 
 RUN apt-get update && \
-    apt-get upgrade -y
+    apt-get upgrade -y && \
+    apt-get autoremove -y && \
+    rm -rf /var/lib/apt/lists/*
 
 # Stage: build assets
 FROM base as build
 
-RUN apt-get install -y make python g++
+ARG BUILD_NUMBER=1_0_0
```

Comment:
I thought we decided we didn't need these two lines as they were in the base?

Reply:
They are used at lines 37,38 - this is what was causing the build to fail.  It seams in a multistage build the ARG's are not available across stages.